Makefile

```diff
@@ -46,8 +46,10 @@ else
 	DEBUGFLAGS=
 endif
 
+GO ?= go
```

Comment:
Any reason for this? 

If so, can you please add this reason to the PR :smile:

Reply:
Added, the makefile previously had tests failing with  go 1.20.  this added non-disruptive support for multiple  Go versions